rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date)
+    @date = date
+
```

Comment:
- [ ] When you are creating a Rental class, you are to pass in a Book object and a Person object as parameters to the initialize method. `def initialize(date, book, person)`. Kindly implement it. 👍